algorithms/C/sorting/counting-sort.c

```diff
@@ -0,0 +1,33 @@
+#include<stdio.h>
+ 
+void counting_sort(int a[],int n,int max)

```

Comment:
If the function doesn't alter the values of `n` and `max` then wouldn't it be better for program to explicitly mark them as constants ? It would not only help in making this function callable with const values but would also avoid any accidental changes in future.

```suggestion
void counting_sort(int a[], const int n, const int max)
```